driver-sync/src/main/com/mongodb/client/http/FindIterator.java

```diff
@@ -0,0 +1,308 @@
+
+    @Override
+    public <A extends Collection<? super TResult>> A into(final A target) {
+        String res = getResponse(this.filter, "ASC", "");
+        JsonParser parser = new JsonParser();
+        JsonArray jsonArray = (JsonArray) parser.parse(res);
+        for (int i = 0; i < jsonArray.size(); i++) {
+            JsonObject jsonObject1 = jsonArray.get(i).getAsJsonObject();
+            Document doc = Document.parse(jsonObject1.toString());
+            target.add((TResult) doc);
+        }
+        return target;
+    }
+
+    @Override
+    public FindIterable<TResult> batchSize(final int batchSize) {
+        return null;
+    }
+
+    @Override
+    public FindIterable<TResult> collation(final Collation collation) {
+        return null;
+    }
+
+    @Override
+    public FindIterable<TResult> comment(final String comment) {
+        return null;
+    }
+
+    @Override
+    public FindIterable<TResult> hint(final Bson hint) {
+        return null;
+    }
+
+    @Override
+    public FindIterable<TResult> hintString(final String hint) {
+        return null;
+    }
+
+    @Override
+    public FindIterable<TResult> max(final Bson max) {
+        return null;
+    }
+
+    @Override
+    public FindIterable<TResult> min(final Bson min) {
+        return null;
+    }
+
+    @Override
+    public FindIterable<TResult> maxScan(final long maxScan) {
+        return null;
+    }
+
+    @Override
+    public FindIterable<TResult> returnKey(final boolean returnKey) {
+        return null;
+    }
+
+    @Override
+    public FindIterable<TResult> showRecordId(final boolean showRecordId) {
+        return null;
+    }
+    /*
+     *  snapshot
+     */
+    @Override
+    public FindIterable<TResult> snapshot(final boolean snapshot) {
+        return null;
+    }
+
+    public static String getResult(final String mainQuery, final String hostURL) throws IOException {
+        URL url = new URL(hostURL);
+        HttpURLConnection httpConn = (HttpURLConnection) url.openConnection();
+        httpConn.setRequestMethod("POST");
+
+        httpConn.setRequestProperty("x-requested-with", "XMLHttpRequest");
+        httpConn.setRequestProperty("content-type", "application/x-www-form-urlencoded");
+
+        httpConn.setDoOutput(true);
+        OutputStreamWriter writer = new OutputStreamWriter(httpConn.getOutputStream());
+        writer.write(mainQuery);
+        writer.flush();
+        writer.close();
+        httpConn.getOutputStream().close();
+
+        InputStream responseStream = httpConn.getResponseCode() / 100 == 2
+                ? httpConn.getInputStream()
+                : httpConn.getErrorStream();
+        Scanner s = new Scanner(responseStream).useDelimiter("\\A");
+        String response = s.hasNext() ? s.next() : "";
+        return response;
+    }
+
+
+    private Query getQuery(final String filter, final String sortDirection) {
+        Query query = new Query();
+        query.appendQueryKeyValue("Service", "MONGO");
```

Comment:
what happens if I want to query other than "GLOBAL", what is this used for

Reply:
Added functionality to set server type and partnered through the connection url, in same way as done with mongoDB connection string. &serverType=GLOBAL&partnerId=0